code/process_sequences.sh

```diff
@@ -13,7 +13,7 @@
 
 if [ $# -eq 0 ]
 then
- echo "There is no file path provided."
+ echo "There needs to be a path to a compressed blast-db file as input for this script."
```

Comment:
But it's not from blast I just stuck it in that directory... sorry for being confusing.

Reply:
You can just say compressed fasta file

Reply:
Gotcha....

Reply:
I put fastx i think it should be find though cause the code will spit out the same thing I think regardless of fasta or fastq I think...

Reply:
just to be extra safe I'll change it to fasta

Reply:
I think you're right, it will work with either because bioawk is smart enough to handle either, but at the moment we're working with fastas.